modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-precache-feature_{context}"]
+= Using the image pre-cache feature
+
+The SNO spoke clusters might have limited bandwidth to the container image registry, which can cause a time-out before the updates are completed. 
+The pre-cache feature allows the required container images to be present on the spoke cluster before the update maintenance time.
+
+[NOTE]
+====
+The maintenance window is not set by the Topology-Aware Lifecycle Operator. You can apply the Topology-Aware Lifecycle Operator CR at the beginning of the maintenance window by manual application or by external automation. The update procedure start as soon as the Operator CR is created.
```

Comment:
```suggestion
The maintenance window is not set by the Topology Aware Lifecycle Operator. You can apply the Topology Aware Lifecycle Operator CR at the beginning of the maintenance window by manual application or by external automation. The update procedure start as soon as the Operator CR is created.
```

Also, we're not setting a maintenance window right now, so it has to be rephrased. The upgrade starts once the enable flag is set to true.